### ISSUE TITLE
[BUG] Comments under forks can't be marked as spoiler

### ISSUE THREAD

**Issue by suic86** [Codewars/codewars.com#821]
Currently there's no Spoiler flag functionality for comments in under forks. This is maybe a feature but I consider it to be bug as these comments can contain spoiler content too.

**Comment by Chrono79** [comment]
I'd add comments in forums need the spoiler flag too. Will this happen anytime soon?

**Comment by kazk** [comment]
No, and the Forum pages will be removed eventually (I'm thinking of replacing with proper forum like [Discourse](https://www.discourse.org/)).
Also, adding spoilers on Forum doesn't really make sense because there's no information to decide who can view it. It'll be just some toggle to show/hide the comment.

**Comment by Chrono79** [comment]
kazk, I see, I didn't think it enough for the forum functionality. My question was meant about the spoiler in the forks, is it planned? Or maybe remove the forks comments from the Dashboard feed?

**Comment by kazk** [comment]
It wasn't planned. But I might do it if it's low effort.
